test/testsuite/testsuite.go

```diff
@@ -142,14 +142,16 @@ func (s *E2ETestSuite) SetupRelayer(ctx context.Context) (ibc.Relayer, error) {
 }
 
 func (s *E2ETestSuite) DeployXCallMockApp(ctx context.Context, port string) error {
+	testcase := ctx.Value("testcase").(string)
+	connectionKey := fmt.Sprintf("connection-%s", testcase)
 	chainA, chainB := s.GetChains()
 	if err := chainA.DeployXCallMockApp(ctx, chains.XCallConnection{
 		KeyName:                Owner,
 		CounterpartyNid:        chainB.(ibc.Chain).Config().ChainID,
 		ConnectionId:           "connection-0", //TODO
```

Comment:
same here

Reply:
As we intend to establish a connection only once for each xCall, I guess it seems reasonable to use a hardcoded connection ID.